SECURITY.md

```diff
@@ -0,0 +1,126 @@
+# Security Policy
+
+## Introduction
+
+Codenotary, September 7th 2022
+
+*This security policy is based on [this template](https://www.cisa.gov/vulnerability-disclosure-policy-template).*
+
+Codenotary is committed to ensuring the security of the global public by protecting their information. This policy is intended to give security researchers clear guidelines for conducting vulnerability discovery activities and to convey our preferences in how to submit discovered vulnerabilities to us.
+
+This policy is addressed to people using the immudb and / or contributing to the project. It describes **what systems and types of research** are covered under this policy, **how to send** us vulnerability reports, and **how long** we ask security researchers to wait before publicly disclosing vulnerabilities.
+
+We encourage you to contact us to report potential vulnerabilities in our systems.
+
+## Authorization

```

Comment:
WDYT if we have the sections in this order?

1) Introduction
2) Scope
3) Supported Versions
4) Reporting a Vulnerability
5) Guidelines
6) Authorization

Just makes it a bit more easier to read. Adding some references on projects I've used

https://github.com/kubernetes/kubernetes/security/policy
https://github.com/vmware-tanzu/pinniped/security/policy
https://github.com/vmware-tanzu/community-edition/security/policy

Reply:
Thanks, I rearranged the doc accordingly and stole some tips from other policies.